eth_pydantic_types/bytes.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, ClassVar, Optional, Tuple, Type
+
+from pydantic_core.core_schema import (
+    CoreSchema,
+    ValidationInfo,
+    bytes_schema,
+    with_info_before_validator_function,
+)
+
+from eth_pydantic_types.hashing import get_hash_examples, get_hash_pattern
+from eth_pydantic_types.hex import HexBytes
+from eth_pydantic_types.serializers import hex_serializer
+from eth_pydantic_types.validators import validate_bytes_size
+
+
+class Bytes(HexBytes):
```

Comment:
we still want this to be called `HexBytes`.. Only at the ABI level do we want to re-export it with a more ABI-like name, if that makes sense.

but this needs to be HexBytes, we are validating hex related stuff about it.